Wikipedia/Code/Controllers/ArticleDescriptionControlling.swift

```diff
@@ -14,8 +21,16 @@ protocol ArticleDescriptionControlling {
     func currentDescription(completion: @escaping (String?) -> Void)
     func errorTextFromError(_ error: Error) -> String
     func learnMoreViewControllerWithTheme(_ theme: Theme) -> UIViewController?
+    var descriptionMaxLength: Int { get }
+    func warningTypesForDescription(_ description: String?) -> ArticleDescriptionWarningTypes
 }
 
 extension ArticleDescriptionControlling {
     var articleDisplayTitle: String? { return article.displayTitle }
+    var descriptionMaxLength: Int { return 90 }
+    
+    func descriptionIsTooLong(_ description: String?) -> Bool {
```

Comment:
We have `descriptionIsTooLong` in the extension, but the "check capitalization of first letter" function lives in a class. Maybe we should standardize this? Either all the check functions can be in here, or none?

Reply:
@mcleinman sounds good, I have moved the capitalization check into the extension. Originally `descriptionIsTooLong` was just code used in both classes, so I moved it to an extension to share code. Since the capitalization logic was only used in one place there was less of a need, but it feels better now that they're matching.